extract_unique_sensors_01.py

```diff
@@ -62,19 +62,19 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(sensors_list):
+def splitting_list(sensor_data = read_sensors(file_name, url)):
```

Comment:
Does this even run?
![](https://media.giphy.com/media/vV81KcJ9Eu15m/giphy.gif)

@amakhaba Please intervene!

Reply:
Surprisingly, yes it does run.

Reply:
Keboloi!